Content.Server/Ghost/GhostSystem.cs

```diff
@@ -248,6 +251,18 @@ public void OnEntityStorageInsertAttempt(EntityUid uid, GhostComponent comp, Ins
             args.Cancel();
         }
 
+        /// <summary>
+        /// When the round ends, make all players able to see ghosts.
+        /// </summary>
+        /// <param name="args"></param>
+        private void OnRoundEnd(RoundEndTextAppendEvent args)
+        {
+            foreach (var component in EntityQuery<EyeComponent>())
+            {
+                component.VisibilityMask |= (uint) VisibilityFlags.Ghost;
```

Comment:
Ew. Make ghostsystem handle this, make it instead modify the ghost's vis, and add a way to effectively globally toggle ghost visibility (maybe a cool command so admins can lower the veil early)

Reply:
Also don't forget to check during ghost startup what the current visibility mode is.